src/client/components/dimension-list-tile/dimensions-renderer.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from "react";
```

Comment:
import React from "react" will do just as well. ;)

Reply:
It does not work in our TS configuration - the project targets ES5 and uses CommonJS modules. We will change it at one point but not as part of this PR.